src/components/ReserveExcelContent.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import Paper from '@material-ui/core/Paper';
+import Container from '@material-ui/core/Container';
+ 
+const ReserveExcelContent = () => {
+    const excelContent = useSelector(state =>
+        state.appReducer.reserveExcel
+    );
+
+    const generateExcelContent = () => {
+        let content = '';
+
+        if(excelContent && excelContent[0]){
+            return (
+                <div>
+                    <br></br>
+                    <Container>
+                        <TableContainer component={Paper}>
```

Comment:
I was hoping a component could do everything, read the excel, generate the editable spreadsheet table and save it.

But that is fine for now.

Reply:
I can add that in if you want. We should create a new issue for this though.

Reply:
I will create the issue in the backlog

Reply:
https://github.com/NASA-PDS/pds-doi-ui/issues/7